sys-libs/libxcrypt/libxcrypt-4.4.19.ebuild

```diff
@@ -52,19 +61,50 @@ src_configure() {
 	multibuild_foreach_variant multilib-minimal_src_configure
 }
 
+get_xcsysroot() {
+	if is_cross; then
+		echo "${EPREFIX}/usr/${CTARGET}"
+	else
+		echo "/"
```

Comment:
We're sure this shouldn't be ${ESYSROOT} in the non-cross case?

It seems wrong for it to be _just_ /.

Reply:
Yeah, it should be, as otherwise we're [overriding what PMS does by default](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-12800012.3.8).